src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -103,17 +103,17 @@ function MoneyRequestAction(props) {
         // If the childReportID is not present, we need to create a new thread
         const childReportID = lodashGet(props.action, 'childReportID', '0');
         if (childReportID === '0') {
-            const participants = _.uniq([props.session.email, props.action.actorEmail]);
-            const formattedUserLogins = _.map(participants, (login) => OptionsListUtils.addSMSDomainIfPhoneNumber(login).toLowerCase());
+            const participantAccountIDs = _.uniq([props.session.accountID, Number(props.action.actorAccountID)]);
```

Comment:
I think `props.action.actorAccountID` is already a number

Reply:
Think we should get rid of the type coercion or keep it just to be safe? 😬